packages/core/src/queries/verification-status.ts

```diff
@@ -0,0 +1,63 @@
+import type { CreateVerificationStatus, VerificationStatus } from '@logto/schemas';
+import { VerificationStatuses } from '@logto/schemas';
+import { convertToIdentifiers } from '@logto/shared';
+import type { CommonQueryMethods } from 'slonik';
+import { sql } from 'slonik';
+
+import { buildInsertIntoWithPool } from '#src/database/insert-into.js';
+import { DeletionError } from '#src/errors/SlonikError/index.js';
+
+const { table, fields } = convertToIdentifiers(VerificationStatuses);
+
+export const createVerificationStatusQueries = (pool: CommonQueryMethods) => {
+  const findVerificationStatusByUserIdAndSessionId = async (userId: string, sessionId: string) =>
+    pool.maybeOne<VerificationStatus>(sql`
+      select ${sql.join(Object.values(fields), sql`, `)}
+      from ${table}
+      where ${fields.sessionId}=${sessionId} and ${fields.userId}=${userId}
+    `);
+
+  const findVerificationStatusesByUserIdAndSessionId = async (userId: string, sessionId: string) =>
+    pool.any<VerificationStatus>(sql`
+      select ${sql.join(Object.values(fields), sql`, `)}
+      from ${table}
+      where ${fields.sessionId}=${sessionId} and ${fields.userId}=${userId}
+    `);
+
+  const insertVerificationStatus = buildInsertIntoWithPool(pool)<
+    CreateVerificationStatus,
+    VerificationStatus
+  >(VerificationStatuses, {
+    returning: true,
+  });
+
+  const deleteVerificationStatusById = async (id: string) => {
```

Comment:
nit: no need for this query? Can reuse the deleteVerificationStatusesByIds.

Reply:
OK, will remove